app/src/main/java/ru/yandex/practicum/contacts/presentation/base/BaseListDiffCallback.java

```diff
@@ -0,0 +1,23 @@
+package ru.yandex.practicum.contacts.presentation.base;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.DiffUtil;
+
+import ru.yandex.practicum.contacts.presentation.main.ContactUi;
+
+public class BaseListDiffCallback <T extends ListDiffInterface<T>> extends DiffUtil.ItemCallback<T>{
+
+    @Override
+    public boolean areItemsTheSame(@NonNull T oldItem, @NonNull T newItem) {
+        return false;
```

Comment:
В этом методе нужно сравнить поверхностно два объекта:
```suggestion
        return oldItem.theSameAs(newItem);
```